stylesheets/_utility.utilities.scss

```diff
@@ -333,3 +333,13 @@ div.no-results {
 .u-margin-bottom-none {
     margin-bottom: 0 !important;
 }
+
+%u-margin-bottom-1,
```

Comment:
What if the name was just 'u-margin-bottom' for this one and 'u-margin-bottom-2x' for the next class? With just 1 and 2 added at the end might not be clear what is their purpose.

Reply:
Yeah, I agree with @nvasileiadis I'd rather we had `u-margin-bottom` and `u-margin-bottom--large` for example. That would make it a little more consistent with other utility classes with modifiers.